components/Home/RecurringEvents.vue

```diff
@@ -1,19 +1,22 @@
 <i18n lang="yaml">
 en:
   announcement: <strong>DWH</strong> is open {count} nights a week
+  announcement-month: Other monthly events at <strong>DWH</strong>
 nl:
   announcement: <strong>DWH</strong> is elke week op {count} avonden open
+  announcement-month: Andere maandelijkse evenement bij <strong>DWH</strong>
```

Comment:
```suggestion
  announcement-month: Andere maandelijkse evenementen bij <strong>DWH</strong>
```

maar misschien beter
```suggestion
  announcement-month: Andere maandelijkse activiteiten bij <strong>DWH</strong>
```

evenement heeft een net iets andere lading in nl imo

Reply:
Mee eens!